### PR TITLE
added PyScript

### PR DESCRIPTION
**Pull request by Sammojr614**
Made the Python Script Lucas Requested

### CODE REVIEW

File: flash.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+import os
+
+#This Should work correctly, if not let me know -Sam
+os.system("avrdude -c usbasp -p t84 -B 20 -U flash:w:target/avr-attiny84/release/semasher.elf")
```
Comment (freylint) [inline flash.py]: I'd like it if you made the parameters to this utility dynamic.

Something like:
```python3
os.system(f"avrdude -c {PROGRAMMER} -p {TARGET} -B {TARGET_CLOCK_MHZ} -U flash:w:{BIN}")
```

Comment (freylint) [inline flash.py]: I'd also like the output of the programmer to show up in the console. This currently just returns a posix style success code.

**Review by freylint — CHANGES_REQUESTED** [review]
Good work. Just a couple more requirements that were failed to be mentioned.

**Comment by Sammojr614** [comment]
I Did the changes, Should be good now!

File: flash.py [hunk 2]
```diff
@@ -1,4 +1,10 @@
 import os
-
+import subprocess
+PROGRAMMER = "sammo"
```
Comment (freylint) [inline flash.py]: Little issue here. This is referring to the micro-controller programmer, as in the way we put code onto the chip. It should be `"usbasp"`. If you'd like to credit yourself please put your name in a comment in the top of the script.

**Review by freylint — CHANGES_REQUESTED** [review]

**Review by freylint — APPROVED** [review]

**Pull request merged**